File: Questao13/Regex.java
import java.util.regex.*;  
public class Regex{  
	public static void main(String args[]){  
		System.out.println("\\.s\\ casa com \"as\"? "+ Pattern.matches(".s", "as")); 
		System.out.println("\\.s\\ casa com \"mk\"? "+Pattern.matches(".s", "mk"));
		System.out.println("\\.s\\ casa com \"mst\"? "+Pattern.matches(".s", "mst"));  
		System.out.println("\\..s\\ casa com \"mas\"? "+Pattern.matches("..s", "mas"));
	}
}

File: Questao13/Regex.py
import re
print("\\.s\\ casa com \"as\"? ")
if re.search(".s","as") != None:
 print("true");
else:
 print("false");
print("\\.s\\ casa com \"mk\"? ")
if re.search(".s","mk") != None:
 print("true");
else:
 print("false");
print("\\.s\\ casa com \"mst\"? ")
if re.search(".s","mst") != None:
 print("true");
else:
 print("false");
print("\\..s\\ casa com \"mas\"? ")
if re.search("..s","mas") != None:
 print("true");
else:
 print("false");
